contents/teams/error-tracking/objectives.mdx

```diff
@@ -0,0 +1,20 @@
+### 🎯 Objective: Human readable stack traces for Python
+
+* Separate ingestion route to produce human readable stack trace from new exceptions
+* Allow customers to upload sourcemaps to content addressable store
```

Comment:
being nosy and reviewing 😊 

I thought content addressable is an optimisation and doesn't need to be a goal at this point

Reply:
```suggestion
* Store customer provided source map files
```